security/docker/nodeagent-sds-cert.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
maybe put expiration duration longer than 1 year ?

Reply:
Update to 10 years.